src/model/profile.ts

```diff
@@ -17,32 +17,96 @@
  */
 import * as Table from 'cli-table3';
 import {HorizontalTable} from 'cli-table3';
-import {Account, NetworkType} from 'nem2-sdk';
+import {Address, EncryptedPrivateKey, NetworkType, Password, SimpleWallet} from 'nem2-sdk';
+
+export interface AddressDTO {
+    address: string;
+    networkType: number;
+}
+
+export interface EncryptedPrivateKeyDTO {
+    encryptedKey: string;
+    iv: string;
+}
+
+export interface SimpleWalletDTO {
+    name: string;
+    network: number;
+    address: AddressDTO;
+    creationDate: string;
+    schema: string;
+    encryptedPrivateKey: EncryptedPrivateKeyDTO;
+}
+
+export interface AccountDTO {
+    simpleWallet: SimpleWalletDTO;
+    url: string;
+    networkGenerationHash: string;
+}
 
 export class Profile {
     private readonly table: HorizontalTable;
 
-    constructor(public readonly account: Account,
-                public readonly networkType: NetworkType,
+    public static createFromDTO(DTO: AccountDTO): Profile {
+        const simpleWallet = new SimpleWallet(
+            DTO.simpleWallet.name,
+            DTO.simpleWallet.network,
+            Address.createFromRawAddress(DTO.simpleWallet.address.address),
+            // @ts-ignore
```

Comment:
why this is ignored?

Reply:
this is a hack. I pushed a PR to the SDK to handle SimpleWallet installation, can clean that up @next SDK release